specs/subresourceintegrity/spec.markdown

```diff
@@ -350,19 +350,29 @@ The following algorithm details these restrictions:
 
 1.  Let <var>request</var> be the request that fetched
     <var>resource</var>.
-2.  If the [mode][fetch-mode] of <var>request</var> is `CORS`,
+2.  If the [response tainting][] of <var>request</var> is `opaque`,
+    return`false`.
```

Comment:
If resource is the response, why not simply check whether it is opaque? It seems weird to use properties of the request after fetching finished.